### DISCUSSION TITLE
Anzeige Einspeisung (Vergütung) in der Mittelachse bei Tag, Woche, Monat, Jahr, Gesamt

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3595] [Ideas]
Ich habe mich in der Stunde plus die es heute gab mit folgender Idee befasst.
In der Anzeige Tag, --> Jahr werden in der Mittelachse von Solectrus u.a. Kosten und Ersparniss angezeigt.
Wäre es nicht auch sinnvoll in dieser Anzeige ober- oder unterhalb der Kosten die Einspeißung in € anzuzeigen ?
Im Dashboard wird es eng, da alle mit runder Grafik angezeigt werden. In der mobilen Anzeige wäre Platz genug.
Was haltet ihr davon ?

![grafik](https://github.com/user-attachments/assets/3c098705-15e1-4fb4-aa2c-829e8e2b1b52)

**Comment by ledermann** [comment]
Danke für den Input, ich bin jedoch eher davon abgeneigt. Es ist meiner Ansicht nach bereits ziemlich viel Information auf einmal in der Strombilanz zu sehen. Ich würde lieber überlegen, was man weglassen oder ganz woandershin schieben könnte ;-)
